test/gtest/uct/test_progress.cc

```diff
@@ -17,6 +17,23 @@ class test_uct_progress : public uct_test {
         uct_test::init();
         m_entities.push_back(create_entity(0));
     }
+
+protected:
+    uct_worker_h worker(unsigned index = 0) {
```

Comment:
Please review code format issues in test_callbackq.cc and in test_progress.cc. There are several issues with brace location.